app/main.py

```diff
@@ -1,6 +1,28 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+URL = "https://api.weatherapi.com/v1/current.json"
+QUERY = {"key": os.getenv("API_KEY"), "q": "Paris"}
```

Comment:
place constant(s) and load_dotenv() into class/function or program start-point to not create redundant imports at the unexpected program places

Reply:
Variable in function should be lowercase